src/Kernel-BytecodeEncoders/EncoderForSistaV1.class.st

```diff
@@ -469,7 +469,9 @@ EncoderForSistaV1 class >> isCreateBlockAt: pc in: method [
 
 	| byte |
 	byte := self nonExtensionBytecodeAt: pc in: method.
-	^ byte = 250
+	^ byte = 250 or: [
+		  (method sourceNodeForPC: pc) in: [ :node | "Clean blocks are created as PushConstant"
```

Comment:
Falling back to the AST here is not that nice as it breaks the abstraction (the idea is that the bytecode level abstractions should be usable without the AST). But we could merge it for now and improve later, this is used only the the debugger which needs the AST anyway.